docs/user-guide/spatial-hashing.ipynb

```diff
@@ -0,0 +1,104 @@
+{
```

Comment:
<p>Should be updated. To keep things simple, maybe we can use our quad hexagon grid, which is a small example grid. </p><p> </p>

<pre class="ql-syntax" spellcheck="false">grid_path = "../../test/meshfiles/ugrid/quad-hexagon/grid.nc"
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/UXARRAY/uxarray/pull/1169/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
It's very tiny

<img width="774" alt="image" src="https://github.com/user-attachments/assets/1edfbc53-3b99-4809-8608-8324a6c1eb64" />


Feel free to keep the MPAS one.

Reply:
Updated to smaller hex grid here : https://github.com/UXARRAY/uxarray/pull/1169/commits/fb4bbf82a0647c262a113b5a5a0dfa99385e0c23